### DISCUSSION TITLE
Where are ScratchAddons settings stored?

### DISCUSSION THREAD

**Discussion by stefanofederici** [ScratchAddons/ScratchAddons#4377] [Help]
Thanks in advance

**Comment by apple502j** [comment]
This is stored in your computer (or, synced from your computer via Google/Firefox account if you're signed in and turned on automatic syncing)

**Comment by iqnite** [comment]
Yes, they're in some hidden JavaScript storage. If you want to get the settings file to export it, you can go to Scratch Addons > Settings > More settings > Import/Export settings
